### ISSUE TITLE
Using multiple accounts is unfriendly and I am not always sure which account I am using

### ISSUE THREAD

**Issue by Mikaela** [siacs/Conversations#3074]
#### General information

* **Version:** 2.2.2
* **Device:** OnePlus 3
* **Android Version:** Stock/OxygenOS 8(?) Oreo
* **Server name:** blesmrt.net
* **Server software:** I don't remember, why server info doesn't tell this?
* **Installed server modules:** partial PEP, everything else server info shows available
* **Conversations source:** PlayStore Beta Channel

#### Steps to reproduce

1. Add multiple accounts to Conversations
2. Look at the main screen
3. Open a contact or MUC

#### Expected result

I am always aware what account I am using.

#### Actual result

I see no separation of what account I am using unless I open contact details or group chat details.

##### My ideas

* When input boxes are empty, they read "Send unencrypted message" or "Send omemo encrypted message". Could "from account@example.net" or "as account@example.net" be added to it?
* In main screen, could there be something small like "account symbol" (a ball, square or triangle) be in front of the contact/MUC? I am taking this idea from Firefox Container Tabs.
* Gajim has collapseable categories for different accounts in the roster and has option for having one chat window per account and then using tabs inside it. Could this behaviour be somehow brought to Conversations main screen?

**Comment by Mikaela** [comment]
@licaon-kter I had this tab automatically restored and noticed that you have thumbs downed this. May I ask why or if you have better ideas or are against some of my ideas or just what thoughts do you have on this?

**Comment by licaon-kter** [comment]
I don't like the "tree view", also I don't quite get the whole problem, so you are logged in to the some chats/MUCs from multiple accounts?

**Comment by Mikaela** [comment]
>  so you are logged in to the some chats/MUCs from multiple accounts?

Yes, my main account is on blesmrt.net, but lately I have also been testing on disroot.org account which has their MUC in bookmarks and yesterday I also added organisation HQ projector machine to XMPP to showcase it and I found it easier to do some tasks on Conversations so I added it to mine temporarily and the result is that I have many duplicates of some rooms and when I open them I am told that the account is disabled while in case of Disroot I wouldn't mind the account being enabled all the time.

There are also other reasons why one would have multiple accounts in teh same MUC such as wanting to lure spammers to message not-moderator connection (I don't know if anyone does that outside of Telegram group) and as there is no sorting of MUCs, I would like to be able to use a separate account for volunteer work which I could then disable as I get too much of it, while having some MUCs where it would be beneficial to be with both accounts.

**Comment by BigGiantHeadsHighCommander** [comment]
On your device don't use the same avatar with each of your accounts.

On Conversations YOU can override a contact's avatar (assumes either not linked to contact provider, or is linked but no contact image provided). With this feature YOU can visually create separation yourself.

**Comment by Olf0** [comment]
While reading this, I couldn't resist:
Being deliberately logged into the same MUC with multiple own accounts simultaneously seems to be a really unusual use case.

**Comment by Olf0** [comment]
@contrapunctus-1, you seem to have misunderstood my statement.
While it is a common use case to use different XMPP accounts (with different avatar pictures) for different MUCs, using different XMPP accounts concurrently for the **same** MUC does not make much sense to me.

**Comment by SamWhited** [comment]
> But that's what is recommended when someone asks for the ability to set a different avatar for each MUC

This seems like a niche use case as well, I'm afraid :)

**Comment by zz-ha-dum** [comment]
This is all the more fun when each of your JID has the same username component

meaty@xmpp.zone
meaty@420chat.me
meaty@conversations.im

**Comment by r4dh4l** [comment]
> If you have the same contact on multiple accounts you are doing something wrong.

Uh. I just try to imagine how a multi account mail client like Thunderbird would feel if I can not see directly which account I'm using.
My respect for Conversations but imho statements like "you are doing something wrong" are not always the ideal way to handle user experience.

> If you have multiple accounts for reliability reasons pick a different provider.

For reliability reasons I pick not a _different_ provider but an _additonal_ provider. Every provider can be unreliable and for this moment it is good to have a second one with a similar roster setup.

> If you have multiple accounts because you like to toy around and test things then I’m sorry but Conversations isn’t trying to cover that use case.

Convincing people of XMPP was always difficult for me and ends up in testing different providers even I found my personla "reliable one". Even just for helping XMPP newcomers feature requested by @Mikaela would be very helpful. Is an extended info text in the text field really such a big deal?

**Comment by licaon-kter** [comment]
Pix-Art now shows the account in chats view under the last message, not pleasing visually since it goes under the icon alignment.

**Comment by gitterspec** [comment]
This is a really important usability gap.  Often I have different accounts interacting with the same bots or support contacts from different services over XMPP.

**Comment by inferenceus** [comment]
> When input boxes are empty, they read "Send unencrypted message" or "Send omemo encrypted message". Could "from [account@example.net](mailto:account@example.net)" or "as [account@example.net](mailto:account@example.net)" be added to it?

This is what I also propose, changing the current "Send an (un)encrypted message" text in the message box to the JID currently in use in the MUC. Since encryption status can be determined by the padlock symbol at the top-right of the screen, there is no need for it to be duplicated into text in the message box.

**Comment by inferenceus** [comment]
@iNPUTmice 

> If you have the same contact on multiple accounts you are doing something wrong.

You think using multiple accounts for redundancy in case of server downtime is a bad thing and shouldn't be done?
